app/javascript/lib/vue/accesibility/index.js

```diff
@@ -0,0 +1,8 @@
+import Vue from "vue";
+
+export function checkAndReportAccessibility() {
+  if (process.env.NODE_ENV === 'development') {
```

Comment:
😮

Reply:
😄

Reply:
I don't like it either, neither to have to call explicitly to the function `checkAndReportAccessibility` in all JS controllers.

What if we create a js package for accessibility and in the layouts we load it only when Rails env == 'development' ??